chrono-tz-build/src/lib.rs

```diff
@@ -555,13 +557,25 @@ pub fn main() {
 
     let mut table = table.build();
     filter::maybe_filter_timezone_table(&mut table);
+    let iana_db_version = detect_iana_db_version();
 
-    let timezone_path = Path::new(&env::var("OUT_DIR").unwrap()).join("timezones.rs");
+    if env::var("CHRONO_TZ_UPDATE_PREBUILT").as_deref() == Ok("1") {
+        let src_dir = env::current_dir().unwrap();
+        let timezone_path = src_dir.join("src").join("prebuilt_timezones.rs");
+        let mut timezone_file = File::create(timezone_path).unwrap();
+        write_timezone_file(&mut timezone_file, &table).unwrap();
+
+        let directory_path = src_dir.join("src").join("prebuilt_directory.rs");
+        let mut directory_file = File::create(directory_path).unwrap();
+        write_directory_file(&mut directory_file, &table, &iana_db_version).unwrap();
```

Comment:
Can we extract the duplicated code out into a function instead?

Reply:
Sure.